Gitlab/two-stage-pipeline-template/hooks/post_gen_project.py

```diff
@@ -0,0 +1,24 @@
+import os
+import shutil
+
+output_dir = "{{ cookiecutter.outputDir }}"
+project_root_dir = os.path.relpath(".", output_dir)
+gitlabFile = ".gitlab-ci.yml"
+assumeRoleFile = "assume-role.sh"
```

Comment:
in python, the variable names need to follow `x_y_z` (snake_case) instead of `xYZ`

Reply:
Got it. I guess I just wrote too much Java lol